study.py

```diff
@@ -27,14 +27,16 @@ def study():
     simCnt = 0
 
     now = datetime.now()
+    if config.deterministic:
+        random.seed("DASsimulator")
+    else:
+        random.seed(str(now).split(".")[1])
```

Comment:
I think random uses current system time or is randomness by default, so you don't need the parameters.